.github/workflows/python-package-conda.yml

```diff
@@ -20,32 +17,32 @@ jobs:
           conda-channels: conda-forge, bioconda, r, defaults
       - name: Setup file system
         run: |
-          sudo mkdir -p /data
+          sudo mkdir -p /data && \
           sudo chmod 777 /data
       - name: Setup environment variables
         run: |
           export PAPYRUS_ACCESS_STRING=mongodb+srv://inigo-montoya:1aBc5678910@testcluster.jm2x3.mongodb.net/test
       - name: Install dependencies
         run: |
-          pip install flake8
-          conda env create --file setup/other_envs/ucsc.yml --name ucsc
-          conda env create -f setup/other_envs/cutadaptenv.yml --name cutadaptenv
-          conda env create --file setup/other_envs/cnmops.yml --name cn.mops
-          conda env create --file setup/environment.yml --name genomics.py3
+          pip install flake8 
+          conda env create --file setup/other_envs/ucsc.yml --name ucsc 
+          conda env create -f setup/other_envs/cutadaptenv.yml --name cutadaptenv 
```

Comment:
Maybe remove all these spaces?

Reply:
Can you explain how you resolved that? 
I guess by removing these line
   defaults:
      run:
        shell: bash -l {0}
        
but why?

Reply:
I read something about github using set -e by default, but not when using a non-default shell, which isprobably what we did here.